node-tests/utils_test.js

```diff
@@ -159,7 +159,7 @@ describe('utils', function () {
       assert.strictEqual(utils.isInlinePrecompileBabelPluginRegistered(plugins), true);
     });
 
-    it('detects when the parallelizable version of the plugin has been installed', function () {
+    it.only('detects when the parallelizable version of the plugin has been installed', function () {
```

Comment:
@ef4 Looks like you left an only
```suggestion
    it('detects when the parallelizable version of the plugin has been installed', function () {
```

Reply:
Good catch.